source/extensions/transport_sockets/proxy_protocol/proxy_protocol.cc

```diff
@@ -92,12 +94,13 @@ Network::IoResult UpstreamProxyProtocolSocket::writeHeader() {
                      result.err_->getErrorDetails());
       if (result.err_->getErrorCode() != Api::IoError::IoErrorCode::Again) {
         action = Network::PostIoAction::Close;
+        didBlock = true;
```

Comment:
Another !=/== discrepancy with Api::IoError::IoErrorCode::Again

Reply:
fixed in c7b20241c5eeed82d8d8ed356a8ac79d2d935c9c